src/main/java/org/ethereum/beacon/discovery/schema/NodeRecordBuilder.java

```diff
@@ -70,6 +75,11 @@ static void addFieldsForUdpAddress(
     fields.add(new EnrField(isIpV6 ? EnrField.UDP_V6 : EnrField.UDP, udpPort));
   }
 
+  static void addCustomField(
+      final List<EnrField> fields, final String fieldName, final Bytes value) {
+    fields.add(new EnrField(fieldName, value));
+  }
```

Comment:
Does this really need to be it's own method?

Reply:
Not really but I wanted to make sure I adhere to the convention.